algebird-core/src/main/scala/com/twitter/algebird/BloomFilter.scala

```diff
@@ -369,12 +369,41 @@ sealed abstract class BF[A] extends java.io.Serializable {
   // Estimates the cardinality of the set of elements that have been
   // inserted into the Bloom Filter.
   def size: Approximate[Long]
+
+  def toBitSet: BitSet
+
+  /**
+   * Compute the Hamming distance between the two Bloom filters
+   * `a` and `b`. The distance is defined as the number of bits that
+   * need to change to in order to transform one filter into the other.
+   */
+  def hammingDistance(that: BF[A]): Int = {
+    assert((this.width == that.width) && (this.numHashes == that.numHashes),
+      "To compute a distance between two Bloom filters they " +
+        "have to be of equal width and the same number of hashes. " +
+        s"A was of width ${this.width} and had ${this.numHashes} hashes, and " +
+        s"B was of width ${that.width} and had ${that.numHashes} hashes.")
+
+    val aSet = this.toBitSet
```

Comment:
Note that this can be made more efficient. BFZero and BFItem can be handled specially and underlying compressed bitmap of BFSparse has a xorCardinality method that avoids intermediate allocations.